src/buildercore/bootstrap.py

```diff
@@ -175,19 +174,27 @@ def is_resourcing():
 
 def remove_topics_from_sqs_policy(policy, topic_arns):
     """Removes statements from an SQS policy.
-
     These statements are created by boto's subscribe_sqs_queue()"""
 
     def for_unsubbed_topic(statement):
+        # `statement` looks like:
+        # {u'Statement': [{u'Action': u'SQS:SendMessage',
+        #   u'Condition': {u'StringLike': {u'aws:SourceArn': u'arn:aws:sns:us-east-1:512686554592:bus-profiles--end2end'}},
+        #   u'Effect': u'Allow',
+        #   u'Principal': {u'AWS': u'*'},
+        #   u'Resource': u'arn:aws:sqs:us-east-1:512686554592:annotations--end2end',
+        #   u'Sid': u'5a1770151e27027c1f43d7f3c968fc10'}],
+        # u'Version': u'2008-10-17'}
         return statement.get('Condition', {}).get('StringLike', {}).get('aws:SourceArn') in topic_arns
 
     policy['Statement'] = list([s for s in policy.get('Statement', []) if not for_unsubbed_topic(s)])
     if policy['Statement']:
         return policy
+    # TODO: unreachable code, policy['Statement'] will always be something
```

Comment:
probably got this way after several rounds of running a full update, and tweaking after the failure